<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculationInfo</class>
 <widget class="QWidget" name="CalculationInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>487</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>11</pointsize>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>26</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="labelFile">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Results file path</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>182</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="labelCoincidences">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Total coincidences: 0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>31</height>
      </size>
     </property>
     <property name="maximum">
      <number>0</number>
     </property>
     <property name="value">
      <number>-1</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>166</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>513</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="buttonPause">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="buttonStop">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
